src/node/app/update.ts

```diff
@@ -27,7 +18,7 @@ export interface LatestResponse {
 }
 
 /**
- * Update HTTP provider.
+ * HTTP provider for checking updates (does not download/install them).
```

Comment:
Let's clarify this exists due to history. I think it'd make more sense to just check for updates client side.

Reply:
When we have a dashboard I was thinking it would be helpful to show whether your version of code-server was out of date. Currently all the HTML is generated server-side so if we keep that paradigm we'll need this to be server-side as well. 

The other thing is that having this as an endpoint lets the checking code be shared between vscode and code-server without having to copy it or bundle it into both code-server and vscode, or something like that, if that makes sense.

Reply:
Makes sense.